temporallib/client/client.py

```diff
@@ -47,6 +56,9 @@ async def connect(
         retry_config: Optional[RetryConfig] = None,
         rpc_metadata: Mapping[str, str] = None,
         identity: Optional[str] = None,
+        lazy: bool = False, 
```

Comment:
i don't see the lazy thing passed to the connect later

Reply:
Added, thanks for that!